dev-libs/libxml2/libxml2-2.12.5.recipe

```diff
@@ -44,6 +44,7 @@ fi
 REQUIRES="
 	haiku$secondaryArchSuffix
 	lib:libz$secondaryArchSuffix
+        lib:libiconv
```

Comment:
The indentation looks wrong (did you use spaces instead of a tab?). Also, this is missing `$secondaryArchSuffix`.